src/test/scala/io/citrine/lolo/util/LoloPyDataLoaderTest.scala

```diff
@@ -0,0 +1,111 @@
+package io.citrine.lolo.util
+import io.citrine.lolo.TestUtils
+import io.citrine.lolo.learners.RandomForest
+import io.citrine.lolo.stats.functions.Friedman
+import org.junit.Test
+
+class LoloPyDataLoaderTest {
+
+  private def getData: Array[Byte] = {
+    val stream = getClass.getClassLoader.getResourceAsStream("4by4double.bytes")
+    val byteArray = new Array[Byte](128)
+    stream.read(byteArray)
+    byteArray
+  }
+
+  @Test
+  def testFeatureArray(): Unit = {
+    val data = LoloPyDataLoader.getFeatureArray(getData, 4, bigEndian = false)
+    assert(data.length == 4)
+    assert(data.head.length == 4)
+    assert(Math.abs(data.head(1) - 1) < 1e-6, "Array read in wrong order")
+  }
+
+  @Test
+  def test1DArrayDouble(): Unit = {
+    val data = LoloPyDataLoader.get1DArray(getData, getFloat = true, bigEndian = false).asInstanceOf[Seq[Double]]
+    assert(data.length == 16)
+    assert(Math.abs(data(1) - 1) < 1e-6, "Array read badly")
+  }
+
+  @Test
+  def test1DArrayInteger(): Unit = {
+    val data = LoloPyDataLoader.get1DArray(getData, getFloat = false, bigEndian = false).asInstanceOf[Seq[Int]]
+    assert(data.length == 32)
+    assert(data.take(3).sum == 0) // First 3 values are all 0
+    assert(data(3) == 1072693248) // Fourth is a big one
+  }
+
+  @Test
+  def testZipping(): Unit = {
```

Comment:
Does this need to be tested?

Reply:
My suggestion would be to remove it, but I'd ask that you at least comment that it is just testing zip and not something more subtle.  If I were reading this code, I'd probably spend a while trying to figure out what it was testing since I'd assume it wasn't just testing zip.

Reply:
Makes sense. I'll remove it.